voxblox/test/test_fast_vs_baseline_cast_ray.cc

```diff
@@ -0,0 +1,63 @@
+#include <random>
+
+#include <eigen-checks/entrypoint.h>
+#include <eigen-checks/gtest.h>
+#include <gtest/gtest.h>
+
+#include "voxblox/core/tsdf_map.h"
+#include "voxblox/integrator/integrator_utils.h"
+#include "voxblox/integrator/integrator_utils_fast.h"
+#include "voxblox/simulation/sphere_simulator.h"
+
+using namespace voxblox;  // NOLINT
+
+class FastTsdfCastRayTest : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    sphere_points.clear();
+
+    constexpr double kMean = 0;
+    constexpr double kSigma = 0.05;
+    constexpr int kNumPoints = 100;
+    constexpr double kRadius = 1.0;
+
+    sphere_sim::createSphere(kMean, kSigma, kRadius, kNumPoints,
+                             &sphere_points);
+
+    origin = Point(0., 0., 0.);
```

Comment:
yes, you are right, for the benchmark, having a single camera origin is ok, but for the unit tests it isn't.